### PR TITLE
Add the Troubleshooting Guide

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@shenli @siddontang @c4pt0r  PTAL.

**Comment by siddontang** [comment]
LGTM

File: trouble-shooting.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
++ [Open too many files](#open-too-many-files)
++ [Database access times out and the system load is too high](#database-access-times-out-and-the-system-load-is-too-high)
+
+## Cannot connect to the database
+
+1. Make sure all the services are started, including `tidb-server`, `pd-server`, and `tikv-server`.
+2. Use the `ps` command to check if all the processes are running.
+	+ If a certain process is not running, see the following corresponding sections to diagnose and solve the issue.
+	+ If all the processes are running, check the `tidb-server` log to see if the following messages are displayed:
+		+ InfomationSchema is out of date: This message is displayed if the `tikv-server` cannot be connected. Check the state and log of `pd-server` and `tikv-server.
```
Comment (shenli) [inline trouble-shooting.md]: Miss  ` at the end.

**Comment by shenli** [comment]
LGTM

**Pull request merged**